### PR TITLE
[WebPubSub] Update send methods signatures

### PR DESCRIPTION
**Pull request by xingsy97**
# Description
## Diff with latest version (a beta version, `1.1.0b1`)
1. Add three override signatures for send* methods, `application/json`, `text/plain`, and `application/octet-stream`.
2. Add union type to parameter `content` for send* methods
3. Add `groups` parameter to `get_client_access_token`

Note: the next two items were already included in latest version `1.1.0b1`, but will be shown in the diff with stable version `1.0.1`

4. Add new parameter `filter` for send* methods
5. Add new methods `remove_connection_from_all_groups`

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py [hunk 1]
```diff
@@ -126,9 +126,73 @@ def get_client_access_token(self, **kwargs: Any) -> JSON:
 
     get_client_access_token.metadata = {"url": "/api/hubs/{hub}/:generateToken"}  # type: ignore
 
+    @overload
+    def send_to_all(  # pylint: disable=inconsistent-return-statements
+        self, message: Union[str, JSON], *, excluded: Optional[List[str]] = None, filter: Optional[str] = None, content_type: Optional[str] = "application/json", **kwargs: Any
```
Comment (msyyc) [inline sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py:131]:
```suggestion
        self, message: JSON, *, excluded: Optional[List[str]] = None, filter: Optional[str] = None, content_type: Optional[str] = "application/json", **kwargs: Any
```

File: sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py [hunk 2]
```diff
@@ -126,9 +126,73 @@ def get_client_access_token(self, **kwargs: Any) -> JSON:
...
+         messages. Default value is None.
+        :paramtype filter: str
+        :keyword content_type: The content type of the payload. Default value is None. Allowed values are 'application/json', 'application/octet-stream' and 'text/plain'
+        :paramtype content_type: str
+        :return: None
+        :rtype: None
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+
+    def send_to_all(  # pylint: disable=inconsistent-return-statements
```
Comment (msyyc) [inline sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py:151]: lack `@overload`

File: sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py [hunk 3]
```diff
@@ -126,9 +126,73 @@ def get_client_access_token(self, **kwargs: Any) -> JSON:
...
```
Comment (msyyc) [inline sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py:172]: lack `@overload`

File: sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py [hunk 4]
```diff
@@ -201,9 +265,75 @@ def send_to_all(  # pylint: disable=inconsistent-return-statements
         if cls:
             return cls(pipeline_response, None, {})
 
+    @overload
+    def send_to_user(  # pylint: disable=inconsistent-return-statements
+        self, user_id: str, message: Union[str, JSON], *, filter: Optional[str] = None, content_type: Optional[str] = "application/json", **kwargs: Any
```
Comment (msyyc) [inline sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py:270]: same with https://github.com/Azure/azure-sdk-for-python/pull/34855/files#r1531725873

File: sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py (hunk 1 above)
Comment (msyyc) [reply sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py:131]: There are also similar issue in other places and please change them together.

File: sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py [hunk 5]
```diff
@@ -126,9 +126,75 @@ def get_client_access_token(self, **kwargs: Any) -> JSON:
 
     get_client_access_token.metadata = {"url": "/api/hubs/{hub}/:generateToken"}  # type: ignore
 
+    @overload
+    def send_to_all(  # pylint: disable=inconsistent-return-statements
```
Comment (msyyc) [inline sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py:130]: Better to add these changes in CHANGELONG.md

Comment (xingsy97) [reply sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py:130]: @msyyc Latest commit updated changelog.md and corrected types for parameter `content` in comments of `send*` methods. If it's OK, please help approve & merge.

**Review by msyyc — APPROVED** [review]

**Comment by xingsy97** [comment]
@msyyc could you help merge this PR?

**Comment by xingsy97** [comment]
Hi @msyyc , architect told me new API view is ok for him to approve. Please help merge this request.

**Pull request merged**